README.md

```diff
@@ -18,20 +18,15 @@ For more information, see:
 
 ## Installation
 
-First install [Go 1.11](https://golang.org/doc/install) or later and [dep](https://golang.github.io/dep/docs/installation.html). Note that dep requires an explicit GOPATH to be set.
+First install [Go 1.11](https://golang.org/doc/install) latest version (1.11.4) or later.
+Note that prebid-server is using [Go modules](https://blog.golang.org/using-go-modules).
```

Comment:
Should probably add a note here to set `GO111MODULE=on` if you're using Go version <1.13 and are inside `GOPATH`. We can remove that after we stop supporting Go versions <1.13

Reply:
Yes ! I'll do :)